webpack.config.js

```diff
@@ -10,6 +10,12 @@ const dev = mode === 'development';
 const alias = { svelte: path.resolve('node_modules', 'svelte') };
 const extensions = ['.mjs', '.js', '.json', '.svelte', '.html'];
 const mainFields = ['svelte', 'module', 'browser', 'main'];
+const fileLoaderRule = {
+	test: /\.(png|svg|jpg|gif)$/,
```

Comment:
Are there other file extensions that would make sense here? Probably at the very least `.jpeg` as well, but there are likely others. Could we instead have a test that matches things inside `src/node_modules/images` to handle whatever type of file people choose to throw in there, and to also make this more in line with what's happening in the Rollup config?

Reply:
`avif` would be good too https://blog.cloudflare.com/generate-avif-images-with-image-resizing/

Reply:
and `webp`